### PR TITLE
Update README and CODEOWNERS

### LINKED ISSUE #217: Add Setup Script to Readme
In #216 I created a `setup.sh` script to simplify installing all packages needed for Omnibus. Update `README.md` to advertise this script so new devs can use it

### LINKED ISSUE #220: Update Codeowners
Add @BigLiban, @patrick-gu and @Lucifersan to `codeowners`

### PR DESCRIPTION
**Pull request by Darth-Raazi**
## Description

- Updated `README.md` to advertise `setup.sh` script
- Added Mohamed, Patrick and Elizabeth to `CODEOWNERS`

This PR closes #217 and closes #220.

- - -
This change is [Reviewable](https://reviewable.io/reviews/waterloo-rocketry/omnibus/221)

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,5 +1,4 @@
 # This is a CODEOWNERS file to indicate which people are responsible for this repository
 # These people will also be automatically requested for review on PRs
 
-* @waterloo-rocketry/default-reviewers @Darth-Raazi @BluCodeGH @KavinSatheeskumar
-
+* @waterloo-rocketry/default-reviewers @Darth-Raazi @BigLiban @patrick-gu @Lucifersan
```
Comment (BigLiban) [inline .github/CODEOWNERS:4]: Add Saad in here too as he would to like to also be requested for review on PRs too

**Review by BigLiban — CHANGES_REQUESTED** [review]
Everything looks good Ozayr, no issues at all. There's only one comment to look at, but that is it.

**Review by Darth-Raazi — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/omnibus/221)* status: 0 of 3 files reviewed, 1 unresolved discussion (waiting on @BigLiban, @BluCodeGH, @jeffrey-huang-yz, @KavinSatheeskumar, @Lucifersan, @patrick-gu, and @saads312)

___
*[`.github/CODEOWNERS` line 4 at r1](https://reviewable.io/reviews/waterloo-rocketry/omnibus/221#gh-1606956808:-NyMiWzW9wXqbc4JDXJE:bb6un1j) ([raw file](https://github.com/waterloo-rocketry/omnibus/blob/6f5f146b854afe0ab1500a9da4601189535f4d5f/.github/CODEOWNERS#L4)):*
<details><summary><i>Previously, BigLiban (Mohamed Bekdach) wrote…</i></summary><blockquote>

Add Saad in here too as he would to like to also be requested for review on PRs too
</blockquote></details>

Done

**Review by BigLiban — APPROVED** [review]
> Reviewed 2 of 3 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/omnibus/221)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @BluCodeGH, @jeffrey-huang-yz, @KavinSatheeskumar, @Lucifersan, @patrick-gu, and @saads312)

**Review by patrick-gu — APPROVED** [review]
:lgtm:

> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/omnibus/221#-:-NyFnBL3EYPm_EWtLTxo:bnfp4nl)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @BluCodeGH, @jeffrey-huang-yz, @KavinSatheeskumar, @Lucifersan, and @saads312)

**Pull request merged**